UCSD-E4E/Audio_Labeling_System_AID
# Rename user bug fix

Steps to test:

Make a new admin user.
Log in with that user.
In the admin portal, rename the user your are currently log into as to something else
Confirm everything happens normally
Test out other features (changing your current status or changing values for other users)
